src/app/app.component.ts

```diff
@@ -26,11 +22,4 @@ export class AppComponent implements AfterViewInit {
       this.translateService.use(this.translateService.defaultLang)
     }
   }
-
-  ngAfterViewInit() {
-    this.platform.ready().then(() => {
-      this.statusBar.styleDefault()
```

Comment:
I assume this statusbar-library is no longer usable for us. However we need to set the statusbar-style – otherwise at least on iOS the statusbar is invisible while the OS is in dark-mode. Maybe [this capacitor plugin](@capacitor/status-bar) might be of help?

Reply:
Good catch! I added the plugin in ff747c55fea29e058489827c010238c337624683